src/welldev/io/controller/LogIn.java

```diff
@@ -0,0 +1,43 @@
+package welldev.io.controller;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+import welldev.io.DAO.Driver;
+import welldev.io.model.ProductList;
+
+
+public class LogIn extends HttpServlet {
+	
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		String un = request.getParameter("username");
+		String pass = request.getParameter("password");
+		HttpSession session = request.getSession();
+		session.setAttribute("un", un);
+		session.setAttribute("p", pass);
```

Comment:
Why are setting session before verifying username and password?
you should Set session after username and password is matched.